config/locales/en.yml

```diff
@@ -518,28 +515,22 @@ en:
         restart: Restart delivery
         stop: Stop delivery
         unavailable: Unavailable
-      delivery_available: Delivery is available
-      delivery_error_days: Delivery error days
-      delivery_error_hint: If delivery is not possible for %{count} days, it will be automatically marked as undeliverable.
       destroyed_msg: Data from %{domain} is now queued for imminent deletion.
       empty: No domains found.
       known_accounts:
         one: "%{count} known account"
         other: "%{count} known accounts"
       moderation:
         all: All
-        limited: Limited
+        allowed: Allowed
+        noop: Filtered
+        not_limited: Not Limited
+        silenced: Silenced
+        suspended: Suspended
         title: Moderation
-      private_comment: Private comment
-      public_comment: Public comment
       purge: Purge
       purge_description_html: If you believe this domain is offline for good, you can delete all account records and associated data from this domain from your storage. This may take a while.
       title: Federation
-      total_blocked_by_us: Blocked by us
-      total_followed_by_them: Followed by them
-      total_followed_by_us: Followed by us
-      total_reported: Reports about them
-      total_storage: Media attachments
```

Comment:
These were all moved from being rendered server-side to being rendered with javascript.